README.md

```diff
@@ -56,6 +56,7 @@ git pull --recurse-submodules
 - Clone this repository as it's described in the previous section.
 - [Download the Archicad Add-On Development Kit from here](https://archicadapi.graphisoft.com/downloads/api-development-kit) or from [here](https://github.com/GRAPHISOFT/archicad-api-devkit/releases).
 - Generate the IDE project with CMake, and set the following variables:
+  - `AC_VERSION`: The version number of Archicad that the Add-On is built for.
```

Comment:
I won't use past tense here since the add-on has never been built at this point. Maybe it's better to write: Target Archicad version (e.g. "27").